manifests/d/docmirror/dev-sidecar/1.8.1/docmirror.dev-sidecar.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: docmirror.dev-sidecar
+PackageVersion: 1.8.1
+PackageLocale: en-US
+Publisher: Greper
+PublisherUrl: https://github.com/docmirror
+PublisherSupportUrl: https://github.com/docmirror/dev-sidecar/issues
+Author: docmirror
+PackageName: dev-sidecar
+PackageUrl: https://github.com/docmirror/dev-sidecar
+License: MPL-2.0
+LicenseUrl: https://github.com/docmirror/dev-sidecar/blob/HEAD/LICENSE
+CopyrightUrl: https://github.com/docmirror/dev-sidecar/blob/master/LICENSE
+ShortDescription: proxy tool for github
+Tags:
+- electron
+- fastgit
+- git
+- github
+- internet
+- mitmproxy
+- proxy
+ReleaseNotes: "一、更新内容：\n【新特性】\n- 大家期待已久的 「暗色主题」 来啦！！！ #292\n- 实现篡改猴（Tampermonkey，别名：油猴）插件脚本的几个 GM_xxx 方法，并参照篡改猴浏览器扩展界面开发了可操作界面，以解锁 github油猴脚本 的所有功能。 #300另外，篡改猴插件地址，可以自定义配置了 #301\n插件新增快捷键：\nCtrl+Alt+H：隐藏插件图标\nCtrl+Alt+S：显示插件图标\n\n- script 拦截器，支持配置 绝对地址 和 相对地址，使该拦截器不再受限于DS的发版了 #294另外提交了 PR #298 简化了该新特性的配置。\n新增一个分支 scripts，用于提供最新的辅助脚本（如：Github油猴脚本），方便用户结合 script 拦截器的该新特性，使用到最新版本的脚本，请访问该分支首页查看配置方法。\n\n- 新增 恢复出厂设置 功能，即：删除/备份 用户个性化配置，重新加载 默认配置 和 远程配置 #291注：目前存在的问题：远程配置的地址也会被重置\n\n- 新增 requestReplace 拦截器，目前暂时只支持替换请求头 #297注：目前已用于辅助 script 拦截器配置绝对地址和相对地址\n\n- 新增 responseReplace 拦截器，目前暂时只支持替换响应头 #295 注：目前已用于辅助 script 拦截器配置绝对地址和相对地址\n\n- InsertScriptMiddleware.js 支持编码方式 deflate 和 br，不再只有 gzip 了 #299\n- 检查更新功能调整：#303\n    - 新版本数据，通过Github API来获取： https://api.github.com/repos/docmirror/dev-sidecar/releases\n    - 自动检查更新功能，可在设置页面关闭\n    - 是否忽略预发布版本，可配置化（只对正式版本生效）\n【scripts 分支】\n- 新增油猴插件的实现：tampermonkey.js （目前实现了部分 GM_xxx 方法，以及可操作界面，参照 #300 中的图片）\n- 新增Github油猴脚本：GithubEnhanced-High-Speed-Download.user.js（版本：2.5.21）\n【问题修复】\n- 修复 OPTIONS.js 在 OPTIONS 请求伪造成功后，实际请求的响应头却不含 Access-Control-Allow-Origin: *，导致跨域请求虽然成功但其响应内容被客户端拒绝的问题 \n- 修复内置Github油猴脚本一处兼容性BUG 5d61e4d\n【优化】\n- 用户个性化配置排在最前面（即恢复到 1.7.3 版本的排序方式） \n- 封装 JSON 解析代码，所有 JSON 的解析使用 JSON5 进行解析 \n- 将App配置保存到 running.json 中，方便用户在提Issue时，了解用户的所有配置信息 \n- 软件安装时间和证书安装时间，使用 yyyy-MM-dd HH:mm:ss.SSS 格式，而不是时间戳 \n- 优化是否绝对地址的判断逻辑，避免误判 \n- 内置Github油猴脚本升级：v2.5.19 -> v2.5.21 \n- 多处日志优化\n【配置】\n- 修改油猴插件为DS仓库中 scripts 分支下的脚本文件 /tampermonkey.js  （#301）\n- 修改github油猴脚本为DS仓库中 scripts 分支下的脚本文件 /GithubEnhanced-High-Speed-Download.user.js  （\n- 优化一处拦截配置：移除一个github的重定向配置，因为代理地址不支持该资源的代理 \n- 移除两处拦截配置：移除github.com的下载加速重定向，因为已经有油猴脚本提供高速下载链接了 \n- 新增一处拦截配置：仓库内的图片，增加缓存配置 #290\n- 新增一处拦截配置：仓库内的JS文件，重定向改为代理，方便script拦截器使用 \n- 优化系统代理配置，移除两个域名，另新增2个WPS域名 \n- 将部分github加速源的域名，添加到系统代理排除列表中 \n【内容】\n- 更新版权信息：版权时间、贡献者姓名 \n二、安装前请仔细阅读安装说明（可以加qq群文件下载安装包）\n注：暂时只发布 Windows 操作系统下的安装包，其他操作系统近期会陆续打包上传，暂时先使用 1.7.3 版本吧。\n|平台|安装说明 |\n|---|---|\n| 【Windows】 | 下载后提示无法验证发行者时，选择保留即可 注意：开着ds重启电脑会导致无法上网，你可以再次打开ds，然后右键小图标退出ds即可。更多说明|\n|  【Mac】 |安装时提示无法验证开发者时，请先取消然后去系统偏好设置->安全与隐私->下方已阻止使用DevSidecar选择仍要打开 |\n|  【Ubuntu】 | 安装说明|\n|【其他Linux】|  |三、Full Changelog\nhttps://github.com/docmirror/dev-sidecar/compare/v1.8.0...v1.8.1"

```

Comment:
I forgot to mention this before approving, but could the release notes be in the localization file's language?